src/coredump/coredump.c

```diff
@@ -800,7 +800,7 @@ static int submit_coredump(
                 /* We cannot log to the journal, so just print the message.
                  * The target was set previously to something safe. */
                 assert(startswith(core_message, "MESSAGE="));
-                log_dispatch(LOG_ERR, 0, core_message + strlen("MESSAGE="));
+                log_dispatch(LOG_ERR, 0, core_message + STRLEN("MESSAGE="));
```

Comment:
Please drop this patch. STRLEN only makes sense in very narrow circumstances: when the length is used to define a VLA. By using STREN() we avoid VLA creation and get a normal stack allocation. In all other circumstances, the compiler replaces strlen on a constant by a constant integer, even at `-O0`.